script/deploy/shared/mongo-log-utils.ts

```diff
@@ -498,8 +442,12 @@ export class RecordTransformer {
           optimizerRuns: validData.OPTIMIZER_RUNS,
           timestamp: new Date(validData.TIMESTAMP),
           constructorArgs: validData.CONSTRUCTOR_ARGS,
-          salt: validData.SALT || undefined,
+          // JSON is source of truth - all optional fields default to empty string
+          salt: validData.SALT || '',
           verified: validData.VERIFIED === 'true',
+          solcVersion: validData.SOLC_VERSION || '',
```

Comment:
is there a check that ensures that we do have all required values for both cases (evm/zkevm/tron?)

so in EVM we want solc and evm version and in zkevm we want all three (or am I wrong?). Example:
https://abscan.org/address/0x9BacA4D8aB4328D167C4503a6c2951c70175CaA9#code (all three values exist here (and are required for manual verification)

Reply:
Added checks in 874b2566